src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -9073,6 +9095,13 @@ namespace HVACUnitarySystem {
 		NoLoadHumRatOut = 0.0;
 		OnOffAirFlowRatio = 0.0; //Autodesk:Init Patch to prevent use uninitialized in calls to SimVariableSpeedCoils
 
+		if ( UnitarySystem( UnitarySysNum ).CondenserNodeNum != 0 ) {
+			OutdoorDryBulb = Node( UnitarySystem( UnitarySysNum ).CondenserNodeNum ).Temp;
+		} else {
+			OutdoorDryBulb = OutDryBulbTemp;
+		}
```

Comment:
Just a clarification for me.  `OutdoorDryBulb` should really be renamed to `CondenserInletAirTemp`, right?  Maybe it's a large change, so not in scope here, I'm just wanting to understand why this block wasn't there before.